clients/java/src/main/java/io/camunda/client/impl/command/JobUpdateCommandImpl.java

```diff
@@ -62,10 +62,12 @@ public CamundaFuture<UpdateJobResponse> send() {
 
   @Override
   public UpdateJobCommandStep2 update(final JobChangeset jobChangeset) {
-    httpRequestObject.setChangeset(
-        new io.camunda.client.protocol.rest.JobChangeset()
-            .retries(jobChangeset.getRetries())
-            .timeout(jobChangeset.getTimeout()));
+    final io.camunda.client.protocol.rest.JobChangeset changeset =
+        new io.camunda.client.protocol.rest.JobChangeset();
+    if (jobChangeset != null) {
+      changeset.retries(jobChangeset.getRetries()).timeout(jobChangeset.getTimeout());
+    }
+    httpRequestObject.setChangeset(changeset);
```

Comment:
👍 Nice. I guess it makes sense to make these work with `null` too and "unset" the value.

Reply:
Yes, at least on the client side. As a developer using this API, I expect that providing `null`, no changes are provided in the changeset. No values will change in the backend this way.